projects/addon-doc/languages/belarusian.ts

```diff
@@ -0,0 +1,48 @@
+import {Provider} from '@angular/core';
+import {
+    TUI_DOC_DEMO_TEXTS,
+    TUI_DOC_DOCUMENTATION_TEXTS,
+    TUI_DOC_EXAMPLE_TEXTS,
+    TUI_DOC_MENU_TEXT,
+    TUI_DOC_SEARCH_TEXT,
+    TUI_DOC_SEE_ALSO_TEXT,
+    TUI_DOC_SOURCE_CODE_TEXT,
+} from '@taiga-ui/addon-doc/tokens';
+
+// TODO: drop in v4.0
+export const TUI_DOC_BELARUSIAN: Provider[] = [
```

Comment:
why we need that in main branch?

Reply:
Let's not introduce localization for addon-doc at this point. Otherwise LGTM.

Reply:
Ok, done.